deployment/CP01CNPM_Deploy.csv

```diff
@@ -8,3 +8,12 @@ AR-24,,AR-28,,CP01CNPM-WFP01-02-DOFSTK000,1,1,2017-11-06T14:55:00,2018-04-12T12:
 AR-24,,AR-28,,CP01CNPM-WFP01-03-CTDPFK000,1,1,2017-11-06T14:55:00,2018-04-12T12:53:00,CGMCP-01CNPM-00001,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.13921667,-70.77951667,,104,134,
 AR-24,,AR-28,,CP01CNPM-WFP01-04-FLORTK000,1,1,2017-11-06T14:55:00,2018-04-12T12:53:00,CGMCP-01CNPM-00001,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.13921667,-70.77951667,,104,134,
 AR-24,,AR-28,,CP01CNPM-WFP01-05-PARADK000,1,1,2017-11-06T14:55:00,2018-04-12T12:53:00,CGMCP-01CNPM-00001,CGVEH-CWFPCP-30002,CGINS-PARADK-20542,40.13921667,-70.77951667,,104,134,
+#AR-31,,,,CP01CNPM-RII01-02-ADCPTG010,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,,CGINS-ADCPTG-18596,40.14,-70.77,,104,134,update post-deployment
+#AR-31,,,,CP01CNPM-SBS01-00-STCENG000,2,1,2018-10-20T00:00:00,,CGMCP-01CNPM-00002,,CGCON-BSTC01-00013,40.14,-70.77,,0,134,update post-deployment
```

Comment:
S/N on config sheet SWE013. Should ID be CGCON-BSTC01-WE013?

Reply:
Confirmed WFP sensor S/Ns